.github/workflows/ci_linux_x64_bazel.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: CI - Linux x64 bazel
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/ci_linux_x64_bazel.yml"
+  schedule:
+    # Weekday mornings at 09:15 UTC = 01:15 PST (UTC - 8).
+    - cron: "15 9 * * 1-5"
+  workflow_dispatch:
```

Comment:
We discussed how frequently this job should run [here on Discord](https://discord.com/channels/689900678990135345/689957613152239638/1286058320825942189).

We settled on a compromise for now:
* Run nightly instead of on every commit 
* Look at setting up a remote build cache on the same nginx server that we use for CMake's ccache/sccache storage (using webdav)